Governance.md

```diff
@@ -18,103 +18,163 @@ Specific goals include:
+- Participating in votes (see [section 5.3 PMC decisions](#53-pmc-decisions) below)
+
+Members of the PMC can resign by sending an email to the PMC chair or the PMC private email list.
+
+#### 2.5.1 PMC Positions
+##### 2.5.1.1 PMC Chair
+The “PMC Chair” is a member of the PMC, elected by the PMC to take this role. The chair has the following responsibilities:
+- Ensures that the functions of the PMC are being performed   
+- Ensures that the activities of the BOINC community are in agreement with this document and established procedures
+- Ensures that votes are taken promptly on issues that require votes
+- Schedules monthly meetings of the PMC
+
+The PMC Chair shall be elected for a term of one year.  They can be re-elected to successive terms up to a maximum of three consecutive terms.
+
+##### 2.5.1.2 PMC Secretary
+The "PMC Secretary" is a member of the PMC, appointed and ratified by the PMC to take this role.  The secretary has the following responsibilities:
+- Ensure that items posted in the public PMC mailing list are addressed   
+- Take notes at monthly meetings of the PMC and distribute them to all PMC members within 7 days of the meeting
+  - Must officially record who attends each meeting as part of the minutes
+- Ensure that a public version of the notes is distributed on the public PMC mailing list
+- Monitor and identify PMC members who should be considered for removal due to inactivity (see [2.5.3 Inactive Removal](#253-inactive-removal) below)
+- Maintain the official record of committers, supporters and members of the PMC (see [section 2.6 Official Record](#26-official-record) below)
+- Act as the official vote counter and recorder for all votes taken
+  - An email to the private PMC list must made at the conclusion of each vote stating:
+    - What was the issue voted on
+    - The outcome of the vote
+    - Who voted and how they cast their vote
+  - An email sent to the public PMC list must be made at the conclusion of each vote stating:
+    - What was the issue voted on
+    - The outcome of the vote
+    - Who voted
+  - In the event that the PMC determines a vote is sensitive in nature, then they can decide to not announce the vote publicly.  The PMC is encouraged to do this rarely and with clear justification.
+
+The PMC Secretary is appointed by the Chair and ratified by the PMC to serve in this role.  Another person can be appointed to this position as determined by the PMC Chair.
+
+#### 2.5.2 PMC Elections
+Any member of the PMC can call for an election of the PMC Chair.  When a member of the PMC calls for an election they must specify that the election is either to immediately replace the current Chair or if the newly elected Chair will start their term when the current term expires. 
+
+Elections shall use the election decision process outlined in [section 5.3.2 Election decisions](#532-election-decisions) below.
+
+#### 2.5.3 Inactive Removal
```

Comment:
For "Inactive Removal" , perhaps note "via Special Voting Procedure outlined in 5.3.1

Reply:
Updated to point to the voting process.  Let me know if you have further feedback here.  Thanks.

Reply:
looks good; thx.